### ISSUE TITLE
Feature Request: Alternate Title Locations

### ISSUE THREAD

**Issue by ZachHaber** [emeeks/semiotic#190]
I'd like to be able to have the titles be placed in different locations: I.E. similar to the axis, top, right, left, bottom. I'd also like to have the titles be able to easily be left justified, centered, or right justified. Finally, if at all possible, the ability to provide a separate spacing between the appropriate graph edge and the title that acts outside of the graph margin.
I played around with moving the title of some graphs to the left side and with vertical text to mixed results using this css. The problems that I came across were that wasn't very simple and requires lots of futzing around to get to look decent, and that it places the title after the margin in the graph, which caused it to have problems with the spacing as I have a lot of margin to accommodate oLabels.
```css
.frame-title__g--left {
  transform: translateY(43%);
}
.frame-title__text--left {
  pointer-events: none;
  writing-mode: vertical-lr;
  transform: rotate(180deg);
  text-anchor: middle;
  dominant-baseline: hanging;
}
```

**Comment by emeeks** [comment]
Sure thing, I'll update title to make it more robust for the next release. I have never heard of `dominant-baseline`!

**Comment by emeeks** [comment]
In the next release title will accept (along with string or SVG JSX) an object of pattern:  `{ title: yourStringOrJSXTitle, orient: "left"|"right"|"top"|"bottom" }`

**Comment by emeeks** [comment]
Available in 1.9.3

**Comment by cvkline** [comment]
Just as a caution:   `dominant-baseline` is not supported in IE11.
